tests/rust/test_api/Cargo.toml

```diff
@@ -11,7 +11,7 @@ workspace = true
 
 [dependencies]
 re_log = { workspace = true, features = ["setup"] }
-rerun = { path = "../../../crates/rerun", features = ["web_viewer"] }
+rerun = { path = "../../../crates/rerun", features = ["clap", "web_viewer"] }
```

Comment:
not seeing any corresponding code changes, was it broken somehow?

Reply:
Yes, running this test standalone did not compile. 

`scripts/ci/rust_checks.py` doesn't cover `tests/rust`, just `examples/rust`